### PR TITLE
Merge new_prepare_data branch with master

### PR DESCRIPTION
**Pull request by sabeenlohawala**

### CODE REVIEW

**Comment by hvgazula** [comment]
overall, one of the most comprehensive codebases I've ever seen. 🫡

File: README.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+## nnUNet
+In this project, we compared the UNet model, which has achieved state-of-the-art results in many medical segmentation models, with the SegFormer model, a newer transformer-based model. We found that the SegFormer model significantly outperformed the original UNet model. The nnUNet has been shown to surpass the original UNet in both 2D and 3D segmentation tasks. Before any firm conclusion can be made about the superior performance of the SegFormer model, the nnUNet performance should be evaluated, as well.
+
+# Contributions
+
+The majority of the code in this repository was written by me ([@sabeenlohawala](https://github.com/sabeenlohawala)). I received invaluable guidance and mentorship on this project, and help with structuring and debugging this code from my advisor Dr. Satrajit Ghosh ([@satra](https://github.com/satra)) and Dr. Harshvardhan Gazula ([@hvgazula](https://github.com/hvgazula)). The earliest version of this code was adapted from a repository by Matthias Steiner ([@batschoni](https://github.com/batschoni)).
```
Comment (hvgazula) [inline README.md:251]: Please remove Dr. in front of my name 🤣 . It's so embarrassing. 🙏

Comment (sabeenlohawala) [reply README.md:251]: is that better?

**Comment by hvgazula** [comment]
Not sure if my previous msg was lost so writing it again.. You are smart lol 🤣. I was happy to see Dr. removed and then noticed Drs. You don't give up so easily huh :). Let's call truce again just like the last time 👊. Fun times..😁

**Pull request merged**